### PR TITLE
Cut release v0.18.0

### PR DESCRIPTION
**Pull request by franknoirot**
Suggested changelog from tool (I'll edit this down while CI runs)

## What's Changed
* Massive app performance improvements by Jess
* Rectangle sketch tool by Frank and Kurt
* Code editor improvements by Jess: autocompletion snippets, find extension, and more
* New panes sidebar by Frank
* New setting to toggle edge highlights
* Migrate to Tauri V2 by Pierre, Paul and Adam
* Allow nightly builds of Modeling App to be installed alongside the standard app by Pierre, Paul and Adam
* New help menu in the bottom right by Frank
* Lots of tests by Kurt, Jess and Frank
* New custom updater (what you're reading this in!) by Pierre
* New app icon by Frank

**Full Changelog**: https://github.com/KittyCAD/modeling-app/compare/v0.17.3...v0.18.0

### CODE REVIEW

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -55,5 +55,8 @@
     }
   },
   "productName": "Zoo Modeling App",
-  "version": "0.17.3"
+  "version": "0.17.3",
```
Comment (franknoirot) [inline src-tauri/tauri.conf.json:58]:
```suggestion
  "version": "0.18.0",
```

File: src-tauri/tauri.conf.json [hunk 2]
```diff
@@ -55,5 +55,8 @@
     }
   },
   "productName": "Zoo Modeling App",
-  "version": "0.17.3"
+  "version": "0.18.0",
```
Comment (franknoirot) [inline src-tauri/tauri.conf.json:58]:
```suggestion
```

File: src-tauri/tauri.conf.json [hunk 3]
```diff
@@ -55,5 +55,7 @@
     }
   },
   "productName": "Zoo Modeling App",
-  "version": "0.17.3"
+  "package": {
```
Comment (franknoirot) [inline src-tauri/tauri.conf.json:58]:
```suggestion
  "version": "0.18.0",
  "package": {
```

**Comment by pierremtb** [comment]
macOS build tests:
- sign in
- open existing project
- create new project
    - sketch on base plane
    - rectangle
    - make it 10x10
    - extrude to 10
    - sketch on top face
    - rectangle
    - make it centered and 4x4
    - extrude to 4
    - export to STL binary, STL ascii, OBJ
    - change project settings
- sign out

image

**Comment by pierremtb** [comment]
Windows 10 build tests:
- sign in
- open existing project
- create new project
    - sketch on base plane
    - rectangle
    - make it 10x10
    - extrude to 10
    - sketch on top face
    - rectangle
    - make it centered and 4x4
    - extrude to 4
    - export to STL binary
    - change project settings
- sign out
- sign back in


image

**Review by pierremtb — APPROVED** [review]

**Pull request merged**